src/core/public/plugins/plugin.test.ts

```diff
@@ -58,13 +59,21 @@ describe('PluginWrapper', () => {
   });
 
   test('`setup` fails if plugin.setup is not a function', async () => {
-    mockInitializer.mockReturnValueOnce({ stop: jest.fn() } as any);
+    mockInitializer.mockReturnValueOnce({ start: jest.fn() } as any);
```

Comment:
Does this test name need to be renamed?

Reply:
Nope this still makes sense. It's testing `setup` with a plugin that doesn't have a setup function. Keeping `stop` in here didn't make sense because it's not required, but now `start` is required.